src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -19,9 +18,8 @@ public Driver create(Driver driver) {
 
     @Override
     public Driver get(Long id) {
-        return driverDao.get(id).orElseThrow(() -> new DataProcessingException(
-                "Couldn't get drivers by id " + id,
-                new RuntimeException()));
+        return driverDao.get(id).orElseThrow(() -> new RuntimeException(
```

Comment:
```suggestion
        return driverDao.get(id).orElseThrow(() -> new DataProcessingException(
```
no need to change exception type, also no need to pass new exception inside `DataProcessingException` constructor, there will be stacktrace anyway